src/api/routes/publisher.js

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (C) 2019  Akhilesh Kumar
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+import {getEntityAliases, getEntityIdentifiers, getPublisherBasicInfo} from '../helpers/formatEntityData';
+import _ from 'lodash';
+import express from 'express';
+import {makeEntityLoader} from '../helpers/entityLoader';
+
+
+const router = express.Router();
+
+const publisherBasicRelations = [
+	'defaultAlias.language',
+	'disambiguation',
+	'publisherType',
+	'area'
+];
+
+const publisherError = 'Publisher not found';
+
+router.get('/:bbid',
+	makeEntityLoader('Publisher', publisherBasicRelations, publisherError),
+	async (req, res, next) => {
+		const publisherBasicInfo = await getPublisherBasicInfo(res.locals.entity);
+		return res.status(200).send(publisherBasicInfo);
+	});
+
+const publisherAliasRelation = ['aliasSet.aliases.language'];
+
+router.get('/:bbid/aliases',
```

Comment:
How to document theses router functions? I am not clear about the comments for the route funtion. Please give the example for one route then I will update accordingly.